selfdrive/updated.py

```diff
@@ -437,7 +439,12 @@ def main() -> None:
       init_overlay()
 
       # ensure we have some params written soon after startup
-      updater.set_params(update_failed_count, exception)
+      updater.set_params(False, update_failed_count, exception)
+
+      if not system_time_valid():
+        wait_helper.sleep(5)
```

Comment:
updater.set_params writes to disk, so let's not do this too often
```suggestion
        wait_helper.sleep(60)
```